fare/grant_staff/api.py

```diff
@@ -0,0 +1,32 @@
+"""api for grant_staff module."""
+from __future__ import absolute_import, print_function
+
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+_datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
+
+
+def grant_staff_permission(user):
+    """Grant staff permission to  a user.
+
+    param string user: the email of the user that obtain staff permission
+    """
+    role = "staff"
+
+    user, role = _datastore._prepare_role_modify_args(user, role)
+
+    # check if the user exist
+    if user is None:
+        return 404
```

Comment:
Everytime we hit an error it is mandatory to log it. I think we have to define a logging system and use it all over the functions implemented, especially in cases like this.

Reply:
See #51